pkg/productsModel/products.go

```diff
@@ -0,0 +1,78 @@
+package productsModel
+
+import (
+	"context"
+	"github.com/appuio/appuio-cloud-reporting/pkg/db"
+	"github.com/jmoiron/sqlx"
+	"github.com/vshn/cloudscale-metrics-collector/pkg/tokenMatcher"
```

Comment:
nitpick: Usually package names are lowercased completely. No hard feeling though
```suggestion
	"github.com/vshn/cloudscale-metrics-collector/pkg/tokenmatcher"
```

Reply:
I've changed that, thanks.